### DISCUSSION TITLE
20241108 Update

### DISCUSSION THREAD

**Discussion by teslasparks301** [rern/rAudio#1804] [General]
My update box says: Please download the latest image file to be able to update. 
This is for the update 20241108. My current version and other specs:

rAudio 20240615
6.6.36-1-rpi aarch64
Raspberry Pi 4B Rev 1.1
BCM2711 • 3.7 GiB
4 x Cortex-A72 @ 1.50 GHz

Does this mean I need to do a reinstall? I hate to sound like such a dunce but I would like to, for various reasons, avoid a reinstall. The latest image I can find is i20240818 which seems rather dated to me.

Can I use the backup made with the current version (20240615)?

mike

**Comment by EarsofTin** [comment]
Maybe this will help:

https://github.com/rern/rAudio/discussions/620

**Reply by teslasparks301** [reply]
Thanks for the reply. Sorry for this belated reply. I thought I'd get an email notification if there was a reply. I didn't and just happened to notice it when I went to the discussions looking for something else.

Now...
I bit the bullet and did a reinstall. I tried my last backup from June but that just seemed to mess things up. So I did the install again and applied the 20241108 update. Then I did a pacman update and then worked my settings. I don't use many of the settings so it was fairly painless. I just don't care for the long DB update since I have a lot of tracks.

mike
